packages/mdc-toolbar/README.md

```diff
@@ -58,7 +58,9 @@ Wrap the items with `mdc-toolbar` class in following way:
 <header class="mdc-toolbar">
   <div class="mdc-toolbar__row">
     <section class="mdc-toolbar__section mdc-toolbar__section--align-start">
-      <a href="#" class="material-icons">menu</a>
+      <span class="mdc-toolbar__menu-icon">
```

Comment:
I think it should be `icon--menu` instead of `menu-icon` and should be applied to the `<a>` tag?